modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -242,126 +241,72 @@ dbms.cluster.discovery.resolver_type=LIST
 dbms.cluster.discovery.endpoints=server01.example.com:5000,server02.example.com:5000,server03.example.com:5000
 dbms.cluster.discovery.v2.endpoints=server01.example.com:6000,server02.example.com:6000,server03.example.com:6000
 ----
-. Restart server01 with the new setting `dbms.cluster.discovery.version=V1_OVER_V2`.
 +
-image:discovery-service/in-place-1-v1-over-v2.png[]
-. Run `SHOW SERVERS` and ensure that all three members are alive.
-. Then repeat steps 2 and 3 for server02 and server03.
-Ensure that they are set to `dbms.cluster.discovery.version=V1_OVER_V2`.
-Restart them sequentially, not in parallel.
+. In Cypher Shell, connect to the `system` database of any server
 +
-image:discovery-service/in-place-23-v1-over-v2.png[]
-
-. Using `bolt://`, connect to the system database of servers 1, 2, 3, and run the following procedure.
-This can be done using via `./cypher-shell -a bolt://localhost:7681 -d system` for example.
+[source, shell, role=nocopy noplay]
+----
+./cypher-shell -a bolt://localhost:7681 -d system
+----
+. Run the procedure:
 +
 [source,cypher]
 ----
 CALL dbms.cluster.showParallelDiscoveryState();
 ----
 +
-They should display "Matching" in the `stateComparison` column.
+The output indicates mode `V1_ONLY`, i.e., only `V1` is running on this server.
```

Comment:
'on this server' or 'on this cluster'?
You say 'on this server' here but the procedure is for moving the entire cluster.

Reply:
This procedure `CALL dbms.cluster.showParallelDiscoveryState();` is used to check the state of a single member. So we need to use the term **server**.  This procedure `CALL internal.dbms.cluster.moveToNextDiscoveryVersion()` moves several members to the next discovery version

Reply:
I think your question is: why do we query the discovery state (CALL dbms.cluster.showParallelDiscoveryState()) of a single server after calling CALL internal.dbms.cluster.moveToNextDiscoveryVersion(); instead of checking the state of other servers?

Our goal is to minimize the number of steps the DB admin needs to take to migrate from V1_ONLY to V2_ONLY. We are quite confident that if the state of a single server matches (i.e., showParallelDiscoveryState returns 'Matching'), the states of the other servers are also likely to match. This is the reason why we don't ask db-admins to check the state of other servers.